pubspec.yaml

```diff
@@ -5,21 +5,13 @@ author: Taylor Anderson <taylor.anderson@workiva.com>
 homepage: https://github.com/Workiva/r_tree
 
 environment:
-  sdk: '>=1.24.3 <3.0.0'
-dependencies:
-  dart2_constant: ^1.0.0
+  sdk: '>=2.9.0 <3.0.0'
+
 dev_dependencies:
   benchmark_harness: any
```

Comment:
We should set this to an actual constraint instead of `any`. Would also be good to add dependency_validator